Java/jni/native_view.cc

```diff
@@ -112,6 +112,15 @@ JNIEXPORT void JNICALL Java_com_couchbase_cbforest_View_delete
         throwError(env, error);
 }
 
+JNIEXPORT jboolean JNICALL Java_com_couchbase_cbforest_View_deleteAtPath
```

Comment:
Doesn't seem useful to return a boolean, since it only ever returns `true`. Change return type to `void`?